packages/kbn-scout/src/playwright/page_objects/maps_page.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { ScoutPage } from '..';
+
+export class MapsPage {
+  // see: https://github.com/elastic/kibana/pull/204607/files#r1932558888
+  readonly directNavigationOrigin = 'maps/map';
+
+  constructor(private readonly page: ScoutPage) {}
+
+  async gotoNewMap() {
+    await this.page.gotoApp(this.directNavigationOrigin);
```

Comment:
nit: I don't think there is value in putting `maps/map` into a constant in the class. In the name of "keeping simple things simple", how about just `await this.page.gotoApp('maps/map');`